openslides/motions/config_variables.py

```diff
@@ -118,11 +116,19 @@ def get_config_variables():
         name='motions_amendments_prefix',
         default_value='A',
```

Comment:
This default value is translated only in JavaScript (motions/site.js). Therefore on server-side (in motions/models.py) while setting the identifier of amentment use the English string "A" instead of the German string "Ä".
Workaround: change config to any string and then back to "Ä". Then "Ä" is used as new prefix.